CHANGELOG.md

```diff
@@ -3,6 +3,26 @@ All notable changes to this project will be documented in this file.
 
 The format is based on [Keep a Changelog](http://keepachangelog.com/) 
 
+## [7.1.2] Preview Release
+### Added
+- Added support for JDK 11
+- Added support for Jacoco in VSTS
+- Updated SQL keywrods in DatabaseMetaData
+- Enabled non-running junit tests
+- Improvements in Database metadata to prevent Statement leaks and enhance Statement caching
+
+### Fixed Issues
+- Fixed slf4j warning message in tests
+- Fixed NullPointerException in logException
+- Fixed intermittent junit lobTest failures
+- Fixed batch insert in Bulk Copy API to respect timeout value
```

Comment:
@peterbae , is this accurate?

Reply:
Sounds a bit wrong. I always make my PR title representative of the change (you can mostly copy-paste the PR title to the changelog, and it'll make sense: https://github.com/Microsoft/mssql-jdbc/pull/817)

I would change this to "Fixed Bulk Copy API for batch insert to respect the timeout value".